addons/l10n_mx/data/account_tax_data.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
   <data>
+    <!-- Tag used only to generate the CFDI (Mexican e-invoice). -->
+    <record id="tag_iva" model="account.account.tag">
+        <field name="name">IVA</field>
+        <field name="applicability">taxes</field>
+    </record>
```

Comment:
Hello @smetl 

Are missing the tags for `ISR` and `IEPS`.

You can see this in [v12](https://github.com/odoo/odoo/blob/12.0/addons/l10n_mx/data/account_tax_data.xml#L5:L16).

The three are required for Mexican localization, not are used the `ISR` and `IEPS`, because only some companies use that taxes.

![Screen Shot 2019-07-30 at 7 14 25 PM](https://user-images.githubusercontent.com/7606656/62174106-4c1e4400-b2fe-11e9-9cbd-63d4ff151b44.png)


Regards

Reply:
@luistorresm But these two tags are not linked to any tax in l10n_mx, right ? If these taxes can be needed, they should probably be created in data instead of expecting people to assign them the right tags, don't you think ? (if so, it should be done in another PR, and coordinated by @jco-odoo )

Reply:
Hello @oco-odoo 

In IEPS are 13 options for taxes, and only some companies use the IEPS, and the ISR could be from 0% to 35%, then are many options. For this reason, We do not add taxes for IEPS or ISR.

![Screen Shot 2019-07-31 at 9 30 04 AM](https://user-images.githubusercontent.com/7606656/62220574-dce43680-b375-11e9-9e4a-71916b7b8669.png)

Example Vauxoo does not have sales for IEPS or ISR.

cc @moylop260

Reply:
@luistorresm Alright, I'm putting these two tags back.

However, for master, I think we should consider another solution when we have time. Also, using indexing like here https://github.com/odoo/enterprise/blame/saas-12.3/l10n_mx_edi/data/3.3/cfdi.xml#L74 (or here, with the fixes: https://github.com/odoo/enterprise/compare/saas-12.3...odoo-dev:saas-12.3-fix-l10n_mx-taxes-las#diff-285983a7a9b586262325a9ce71db0feeR73) to only get the first tag seems totally wrong to me. It could not work in case there are several tags, and using indexing this way is weird. I talked a bit about it with @jco-odoo and we agree on this.